vscode/src/custom-prompts/CommandsController.ts

```diff
@@ -230,9 +237,15 @@ export class CommandsController implements VsCodeCommandsController, vscode.Disp
                 case selectedCommandID === menu_options.config.label:
                     return await vscode.commands.executeCommand('cody.settings.commands')
                 case selectedCommandID === menu_options.chat.label:
-                    return await vscode.commands.executeCommand('cody.inline.new')
+                    return await vscode.commands.executeCommand('cody.inline.new', 'Explain the selected code')
```

Comment:
```suggestion
                    return await vscode.commands.executeCommand('cody.inline.new')
```

this is used for starting a new empty inline chat. if user wants to run the explain command, they can enter `/explain` so i don't think we will need to pre-fill this bit for them. what do you think?